lib/pengine/pe_output.c

```diff
@@ -156,7 +156,8 @@ get_operation_list(xmlNode *rsc_entry) {
         }
 
         /* Ignore notifies and some probes */
-        if (pcmk__str_eq(task, CRMD_ACTION_NOTIFY, pcmk__str_casei) || (pcmk__str_eq(task, "probe", pcmk__str_casei) && (op_rc_i == 7))) {
+        if (pcmk__str_eq(task, PCMK_ACTION_NOTIFY, pcmk__str_casei)
+            || (pcmk__str_eq(task, "probe", pcmk__str_casei) && (op_rc_i == 7))) {
```

Comment:
Do we want a `PCMK_ACTION_PROBE` since it fits into some of the same "task" spots?

Reply:
No, it's purely for logging, not used as an actual action anywhere